pkg/store/bucket.go

```diff
@@ -1610,6 +1629,9 @@ type postingPtr struct {
 // It returns one postings for each key, in the same order.
 // If postings for given key is not fetched, entry at given index will be nil.
 func (r *bucketIndexReader) fetchPostings(keys []labels.Label) ([]index.Postings, error) {
+	start := time.Now()
+	defer r.block.metrics.indexOperationDuration.WithLabelValues(opFetchPostings).Observe(time.Since(start).Seconds())
```

Comment:
Should include the time it takes to fetch from cache? Currently it does. If yes, I would mention it in the metric description.

Reply:
Yes, the idea is to use this metric to also benchmark the performance of the index cache. Either different implementation or different configuration of the cache.

I will mention it in the metric name.